src/components/links/LinkListItem.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import LinkExternal from '../footer/LinkExternal';
+
+const LinkListItem = ({ link }) => {
+	console.log(`🏓🏓🏓 link `, link);
+	if (!link) return null;
+
+	let linkElToRender;
+
+	if (link.isExternal) {
+		linkElToRender = <LinkExternal link={link} rel="noreferrer" target="_blank" />
```

Comment:
We want to remove target =`_blank`

```suggestion
		linkElToRender = <LinkExternal link={link} rel="noreferrer" />
```